src/phoenix/trace/span_evaluations.py

```diff
@@ -152,6 +157,24 @@ def __init_subclass__(
             tuple(sorted(prod)) for prod in product(*cls.index_names.keys())
         )
 
+    def to_parquet(self, path: Union[str, Path]) -> None:
+        table = Table.from_pandas(self.dataframe)
+        table = table.replace_schema_metadata(
+            {
+                **table.schema.metadata,
+                b"openinference": json.dumps({"eval_name": self.eval_name}).encode("utf-8"),
+            }
+        )
+        parquet.write_table(table, path)
+
+    @classmethod
+    def from_parquet(cls: EvaluationsType, path: Union[str, Path]) -> EvaluationsType:
```

Comment:
Shouldn't the cls be part of the file? The caller is not going to know this.

Reply:
`cls` is not visible to the caller. The caller runs `SpanEvaluations.from_parquet(path)`.

Reply:
Oh I see. I was thinking you could have `Evaluations.from_parquet()` which would then know to instantiate a `SpanEvaluations` class by having the file encode the cls.

Reply:
This would work, but `Evaluations` is currently an abstract class that is not user-facing.

Reply:
Being abstract means it can't instantiate. It doesn't mean it isn't user-facing.

```python-console
>>> from abc import ABC
>>> class Test(ABC):
...     @classmethod
...     def test(cls):
...         return cls.__subclasses__()[0]()
...
>>> class MyTest(Test):
...     def answer(self):
...         return 42
...
>>> Test.test().answer()
42
>>> Test.__subclasses__()
[<class '__main__.MyTest'>]
```